app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/vault/VaultSettingsViewModel.kt

```diff
@@ -83,7 +134,13 @@ class VaultSettingsViewModel @Inject constructor(
 data class VaultSettingsState(
     val importUrl: String,
     val isNewImportLoginsFlowEnabled: Boolean,
-)
+    private val showImportActionCard: Boolean,
+) {
+    /**
+     * Should only show the import action card if the import logins feature flag is enabled.
+     */
+    val shouldShowImportCard: Boolean = showImportActionCard && isNewImportLoginsFlowEnabled
```

Comment:
Should this have a `get() =`?

Reply:
So this was working because when we upstate the total state we are creating a new instance of the state with `copy` so this value was getting rewritten each time. For clarity and consistency switched it over to expected syntax for calculated value.